evm-adapters/testdata/CheatCodes.sol

```diff
@@ -294,13 +296,13 @@ contract CheatCodes is DSTest {
 
     function testExpectRevert() public {
         ExpectRevert target = new ExpectRevert();
-        hevm.expectRevert("Value too large");
+        hevm.expectRevert(string("Value too large"));
```

Comment:
I'd say we do `expectRevertBytes` for non-string types, it seems like most of the times it'll be a string and we should try to optimize for that use case

Reply:
Except for custom errors that take parameters in which case it will 100% be bytes - lmk if you disagree, but i dont think its that big of a deal to force people to type their expected reverts - they can basically choose either and itll be fine

Reply:
Yeah I guess my thought was that people prefer using `expectRevert(string)` still, so they'd prefer to not have the explicit `string(...)`? Whereas custom errors have little adoption (could be wrong) so the `expectRevertBytes(0x1234)` wouldn't be a big deal?

Reply:
if anything we should not have `string` and just do `bytes` and `bytes4` because those two basically cover everything. You dont have to typecast a string to bytes if a string method doesnt exist and its longer than 4 bytes

Reply:
I am going to remove string for now